test/functional/tools/catDocker.xml

```diff
@@ -1,9 +1,9 @@
 <tool id="catdc" name="Concatenate datasets (in docker)">
     <description>tail-to-head</description>
     <requirements>
-      <container type="docker">busybox:ubuntu-14.04</container>
+      <container type="docker">bgruening/busybox-bash:0.1</container>

```

Comment:
Since we depended on `bash` we need a new default container.
The source can be found here: https://github.com/bgruening/docker-busybox-bash